docs/proposals-accepted/202206-block-series-deletion-compactor.md

```diff
@@ -0,0 +1,210 @@
+---
+type: proposal
+title: Block Series Deletion with Compactor
+status: accepted
+owner: yeya24
+menu: proposals-accepted
+---
+
+* **Owners:**
+    * @yeya24
+
+## Summary
+
+This design document continues the feature of series deletion for object storage discussed in [proposal](https://thanos.io/tip/proposals-accepted/202012-deletions-object-storage.md/). This proposal is based on the previous work and focuses more on the actual implementation plan and the unaddressed questions from the original proposal like performing the actual deletion in the object storage using Thanos compactor.
+
+## Goals
+
+* Make it more clear about the main workflow of the series deletion process.
+* Perform the actual series deletion using compactor in a scalable way.
+
+## Non-Goals
+
+* Allow setting different retentions per series. This feature can be built on top of this work ideally but it is out of scope of this proposal.
+
+## How
+
+The main workflow of the series deletion will look like this:
+
+1. User create a tombstone file via a CLI tool and the tombstone file is uploaded to the object storage.
```

Comment:
Will the CLI do this, or should it be automated through a CI pipeline?

Reply:
Yes the cli will upload the tombstone file to the bucket directly. Similar to other bucket tools